templates/plugin-test-sample.mustache

```diff
@@ -13,7 +13,7 @@ class SampleTest extends WP_UnitTestCase {
 	/**
 	 * A single example test.
 	 */
-	function test_sample() {
+	public function test_sample() {
```

Comment:
Ditto stuff in the tests directory is ignored so don't think this change is needed.

Reply:
https://wordpress.slack.com/archives/C02RP4T41/p1516728175000577 
that's why I asked 'coz I was not sure if these needed changes

Reply:
Ah I see sorry, so were you getting errors on running phpcs on stuff in the tests directory?